src/timers.h

```diff
@@ -0,0 +1,70 @@
+#ifndef SRC_NODE_TIMERS_H_
+#define SRC_NODE_TIMERS_H_
+
+#if defined(NODE_WANT_INTERNALS) && NODE_WANT_INTERNALS
+
+#include <cinttypes>
+#include "node_snapshotable.h"
+
+namespace node {
+class ExternalReferenceRegistry;
+
+namespace timers {
+class BindingData : public SnapshotableObject {
+ public:
+  BindingData(Environment* env, v8::Local<v8::Object> obj);
+
+  using InternalFieldInfo = InternalFieldInfoBase;
+
+  SERIALIZABLE_OBJECT_METHODS()
+  static constexpr FastStringKey type_name{"node::timers::BindingData"};
+  static constexpr EmbedderObjectType type_int =
+      EmbedderObjectType::k_timers_binding_data;
+
+  SET_NO_MEMORY_INFO()
+  SET_SELF_SIZE(BindingData)
+  SET_MEMORY_INFO_NAME(BindingData)
+
+  static void SetupTimers(const v8::FunctionCallbackInfo<v8::Value>& args);
+
+  static void SlowGetLibuvNow(const v8::FunctionCallbackInfo<v8::Value>& args);
+  static double FastGetLibuvNow(v8::Local<v8::Object> receiver);
```

Comment:
Just noting: this pattern could likely use some documentation around it as we use it more.

Reply:
Yeah I thought about it and then I see that other fast API calls are doing things...slightly differently. So we probably should unify them first, or find a best pattern first? (I am not even sure this is the best pattern but for the purpose of this binding this looks the most sensible to me).

Reply:
Maybe we can add this as an example to https://github.com/nodejs/node/blob/main/doc/contributing/adding-v8-fast-api.md?